src/utils/x1.ts

```diff
@@ -0,0 +1,15 @@
+import { WALLET_CONSTS } from '@soramitsu/soraneo-wallet-web';
+
+export interface X1Widget {
+  sdkUrl: string;
+  widgetId: string;
+}
+
+export class X1Api {
+  public static getWidget(soraNetwork: string): X1Widget {
+    if (soraNetwork === WALLET_CONSTS.SoraNetwork.Prod) {
+      return { sdkUrl: 'https://x1ex.com/widgets/sdk.js', widgetId: 'sprkwdgt-WUQBA5U2' };
```

Comment:
is it the sensitive data? sprkwdgt-WUQBA5U2

Reply:
no, it is rendered in html page on client side to get things working